### ISSUE TITLE
Need to add events to new player account gragh

### ISSUE THREAD

**Issue by giftedprimate** [JWWeatherman/math_bot#749]

**Comment by j-chimienti** [comment]
I'm displaying the events as Gridlines (working in pre prod for me as well)

<img width="1176" alt="screen shot 2019-01-12 at 9 05 50 am" src="https://user-images.githubusercontent.com/16620120/51074127-46eef580-1649-11e9-9529-99bf46e6b24d.png">

### Filter by event date

You can change also change min / max date on signups per day graph by selecting event 

<img width="1167" alt="screen shot 2019-01-12 at 9 07 42 am" src="https://user-images.githubusercontent.com/16620120/51074150-89b0cd80-1649-11e9-800b-300ba5b270b3.png">

**Comment by giftedprimate** [comment]
@j-chimienti I do have the line, I just didn't have the dates set properly. I think it may be a good idea to simplify the date selection to just be between 2 dates. What do you think @JWWeatherman ?

**Comment by JWWeatherman** [comment]
I'm cool with either way. Whatever is easiest for you guys.

**Comment by j-chimienti** [comment]
I agree w/ @giftedprimate, I'll simplify the to just 'between'. dates 'on' and 'before' could be troublesome down the road
